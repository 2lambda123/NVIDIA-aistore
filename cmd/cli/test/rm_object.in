ais create bucket $BUCKET
echo "0123456789" > $OBJECT.txt
ais put $OBJECT.txt $BUCKET/object.txt

ais rm object $BUCKET/object.txt
ais rm object $BUCKET/object.txt // FAIL "doesn't exist"