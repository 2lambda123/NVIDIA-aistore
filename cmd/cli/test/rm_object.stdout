PUT "object.txt" to ais://$BUCKET
deleted "object.txt" from ais://$BUCKET
